english/marshmallow.md

```diff
@@ -731,3 +731,5 @@ Yourself?
 Instead of reading this entire story, you decide to go see the new star wars!
 
 You decide that $1.5 billion is too much to pass up, so you buy a [Powerball ticket](powerball/powerball.md).
+
+Do you have my stapler? [yes](stapler/yes.md) [no](stapler/no.md)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.